packages/x-components/src/design-system/components/input-group/default.scss

```diff
@@ -74,15 +76,22 @@
     flex: 1 1 100%;
```

Comment:
In this line:
```
&:first-child {
      padding-inline-start: var(--x-size-padding-left-input-group-default);
    }
    
 ```
 
   We are adding padding to the first child only. This is doing that when you click on the search input, the placeholder will be focused first, and the cursor appears at the left of the input group but some milliseconds before, the cursor will be in the right position.
   
  I think that we should add the same padding the second child or the placeholder